test/integration/test_create_dss_docker_image.py

```diff
@@ -40,7 +51,7 @@ def dss_docker_container(dss_docker_image):
     container = client.containers.create(
         image=dss_docker_image.image_name,
         name=dss_docker_image.container_name,
-        command="sleep infinity",
+        # command="sleep infinity",
```

Comment:
can be removed

Reply:
Sure I just left it for review. See next push.